docker/run.sh

```diff
@@ -40,115 +40,126 @@ do
 done
 
 if [ -z "${docker_egs}" ]; then
-  echo "Select an example to work with from the egs folder."
-  exit 1
+    echo "Select an example to work with from the egs folder."
+    exit 1
 fi
 
 from_tag="cpu"
 if [ ! "${docker_gpu}" == "-1" ]; then
-  if [ -z "${docker_cuda}" ]; then
-    # If the docker_cuda is not set, the program will automatically 
-    # search the installed version with default configurations (apt)
-    docker_cuda=$( nvcc -V | grep release )
-    docker_cuda=${docker_cuda#*"release "}
-    docker_cuda=${docker_cuda%,*}
-  fi
-  # After search for your cuda version, if the variable docker_cuda is empty the program will raise an error
-  if [ -z "${docker_cuda}" ]; then
-    echo "CUDA was not found in your system. Use CPU image or install NVIDIA-DOCKER, CUDA and NVCC for GPU image."
-    exit 1
-  else
-    from_tag="gpu-cuda${docker_cuda}-cudnn7"
-  fi
+    if [ -z "${docker_cuda}" ]; then
+        # If the docker_cuda is not set, the program will automatically 
+        # search the installed version with default configurations (apt)
+        docker_cuda=$( nvcc -V | grep release )
+        docker_cuda=${docker_cuda#*"release "}
+        docker_cuda=${docker_cuda%,*}
+    fi
+    # After search for your cuda version, if the variable docker_cuda is empty the program will raise an error
+    if [ -z "${docker_cuda}" ]; then
+        echo "CUDA was not found in your system. Use CPU image or install NVIDIA-DOCKER, CUDA and NVCC for GPU image."
+        exit 1
+    else
+        from_tag="gpu-cuda${docker_cuda}-cudnn7"
+    fi
 fi
 
 if [ ! -z "${docker_os}" ]; then
-  from_tag="${from_tag}-${docker_os}"
+    from_tag="${from_tag}-${docker_os}"
 fi
 
 # Check if image exists in the system and download if required
 docker_image=$( docker images -q espnet/espnet:${from_tag} )
 if ! [[ -n ${docker_image}  ]]; then
-  docker pull espnet/espnet:${from_tag}
+    docker pull espnet/espnet:${from_tag}
+fi
+
+if [ ${UID} -eq 0 ] && [ ${docker_user} = true ]; then
+    echo "Warning: Your user ID belongs to root users.
+        Using Docker container with root using instead of User-built container."
```

Comment:
I'm not sure what you mean to say.
Is it like that?
```suggestion
        Using Docker container with root instead of User-built container."
```